transformato/bin/charmm_eval_energy.sh

```diff
@@ -18,4 +15,4 @@ echo 'Path: ' ${path}
 echo 'top: ' ${top}
 echo 'script: ' ${script}
 
-OMP_NUM_THREADS=8 ${CHARMM} top:${top} path:${path} -i ${script}
+charmm_c47_omm_domdecgpu top:${top} path:${path} -i ${script}
```

Comment:
I would prefer that we call a bash variable here that we assume is set in the bash environment --- otherwise it is only usable with this specific executable name.
The `${CHARMM}` keyword was supposed to point to the the bash variable that was set in the environment.